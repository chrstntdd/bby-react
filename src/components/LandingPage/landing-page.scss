.hero-img {
  background-image: url('images/hero-bg-1.jpg');
  background-size: cover;
  background-position: center;
  background-repeat: no-repeat;
  background-attachment: scroll;
}
