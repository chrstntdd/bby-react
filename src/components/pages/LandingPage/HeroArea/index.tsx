import * as React from 'react';
import { Link } from 'react-router-dom';

import './hero-area.scss';

const HeroArea = () => (
  <section id="hero-atf">
    <div className="button-container">
      <Link id="sign-up-button" to="/sign-up">
        <button id="btn-signup">Sign Up</button>
      </Link>

      <Link id="sign-in-button" to="/sign-in">
        <button id="btn-signin">Sign In</button>
      </Link>
    </div>
    <div className="hero-img" />
    <div className="hero-content">
      <h1>Quantified</h1>
      <h2>A streamlined system for efficiently and accurately managing retail inventory.</h2>
    </div>
  </section>
);

export default HeroArea;
