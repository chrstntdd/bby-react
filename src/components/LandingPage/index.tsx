import * as React from 'react';
import { Link } from 'react-router-dom';

import './landing-page.scss';

const navigationButtons =
  'trans-300ms-all z-10 font-semibold rounded-full px-8 py-2 leading-normal bg-transparent border border-grey text-grey hover:border-bby-blue hover:bg-bby-blue hover:text-white m-6';

const LandingPage = () => (
  <div>
    <section className="h-screen w-screen">
      <div className="w-full absolute pin-t z-10 flex justify-center md:justify-end">
        <Link to="/sign-up">
          <button className={navigationButtons}>Sign Up</button>
        </Link>

        <Link to="/sign-in">
          <button className={navigationButtons}>Sign In</button>
        </Link>
      </div>
      <div className="hero-img h-screen w-screen absolute pin" />
      <div className="kinda-center z-10 w-full">
        <h1
          className={
            'text-white italic text-center leading-none whitespace-no-wrap text-5xl md:text-massive tracking-wide mb-2'
          }
        >
          Quantified
        </h1>
        <h2 className="text-white italic text-center leading-none text-xl font-thin">
          A streamlined system for efficiently and accurately managing retail inventory.
        </h2>
      </div>
    </section>
    <section id="how-section">
      <div className="card-container">
        <div className="how-card">
          <img src={'images/atom.svg'} alt={'atom icon'} />
          <h3>Simple</h3>
          <h5>...but powerful</h5>
        </div>
        <div className="how-card">
          <img src={'images/handshake.svg'} alt={'handshake icon'} />
          <h3>Reliable</h3>
          <h5>There when you need it</h5>
        </div>
        <div className="how-card">
          <img src={'images/padlock.svg'} alt={'padlock icon'} />
          <h3>Safe</h3>
          <h5>Without compromises</h5>
        </div>
        <div className="how-card">
          <img src={'images/rocket.svg'} alt={'rocket icon'} />
          <h3>Fast</h3>
          <h5>Designed with speed in mind</h5>
        </div>
      </div>
    </section>
    <section id="about-section">
      <h1>About Quantified</h1>
      <blockquote>
        "A streamlined system for efficiently and accurately managing retail inventory."
      </blockquote>
      <p>
        That sounds great and all, but what does that mean exactly? Imagine for a moment that you
        are in a retail setting. When shipments of products come into your store, they will be
        accompanied by a manifest that notates exactly what products and how many of them were put
        onto the trailer when it left the distribution center. It is the job of one employee to
        verify everything listed on the manifest actually makes it to the store, not an easy or
        quick task by any means.
      </p>
      <p>
        The previous workflow (at least at Best Buy) for counting all of these products involved
        printing out a copy of the aforementioned manifest, then tallying up the products one by
        one. Sounds monotonous and error prone right? That's where Quantified comes in to save the
        day by bearing the responsibility of accounting for everything.
      </p>
      <p>
        Instead of manually counting each product, Quantified allows the user to simply scan the bar
        code of the product and automatically populates all of its relevant information into a row
        in a table. From there, after everything has been scanned, the table can be formatted to
        mirror the manifest and printed for seamless comparison with the manifest for the final
        verification.
      </p>
      <p>
        Additionally, Quantified is built with battle tested and industry leading technology as well
        as the Best Buy API. All together this yields unmatched dependability, speed, and security.
      </p>
    </section>
    <footer>
      <ul>
        <li>
          <Link to={'/sign-up'}>Start here</Link>
        </li>

        <li>
          <Link to={'/sign-in'}>Sign in</Link>
        </li>

        <li>
          <a href={'https://developer.bestbuy.com/api-status'} target="_blank">
            Official Best Buy API Status
          </a>
        </li>

        <li>
          <a href="https://github.com/chrstntdd" target="_blank">
            Created by Christian Todd
          </a>
        </li>
      </ul>
    </footer>
  </div>
);

export default LandingPage;
