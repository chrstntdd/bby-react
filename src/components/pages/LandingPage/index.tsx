import * as React from 'react';

import HeroArea from './HeroArea';
import HowSection from './HowSection';
import AboutSection from './AboutSection';
import Footer from './footer';

const simple = 'images/atom.svg';
const reliable = 'images/handshake.svg';
const secure = 'images/padlock.svg';
const fast = 'images/rocket.svg';

export default class LandingPage extends React.Component {
  constructor(props) {
    super(props);
    this.state = {
      footerData: [
        {
          name: 'Start here',
          link: '/sign-up'
        },
        {
          name: 'Sign in',
          link: '/sign-in'
        },
        {
          name: 'Official Best Buy API Status',
          link: 'https://developer.bestbuy.com/api-status'
        }
      ],
      cardData: [
        {
          name: 'simple',
          img: simple,
          desc: '...but powerful'
        },
        {
          name: 'reliable',
          img: reliable,
          desc: 'There when you need it'
        },
        {
          name: 'safe',
          img: secure,
          desc: 'Without compromises'
        },
        {
          name: 'fast',
          img: fast,
          desc: 'Designed with speed in mind'
        }
      ]
    };
  }
  render() {
    return (
      <div className="landing-page">
        <HeroArea />
        <HowSection cardData={this.state.cardData} />
        <AboutSection />
        <Footer footerData={this.state.footerData} />
      </div>
    );
  }
}
