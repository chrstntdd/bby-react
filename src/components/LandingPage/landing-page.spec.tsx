import * as React from 'react';
import { shallow } from 'enzyme';

import LandingPage from './';

describe('<LandingPage/>', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<LandingPage />);

    expect(wrapper).toMatchSnapshot();
  });
});
