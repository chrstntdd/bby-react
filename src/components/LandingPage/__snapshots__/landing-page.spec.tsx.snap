// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LandingPage/> should render without crashing 1`] = `
<div>
  <section
    className="h-screen w-screen"
  >
    <div
      className="w-full absolute pin-t z-10 flex justify-center md:justify-end"
    >
      <Link
        replace={false}
        to="/sign-up"
      >
        <button
          className="z-10 font-semibold rounded-full px-8 py-2 leading-normal bg-transparent border border-grey text-grey hover:border-bby-blue hover:bg-bby-blue hover:text-white m-6"
        >
          Sign Up
        </button>
      </Link>
      <Link
        replace={false}
        to="/sign-in"
      >
        <button
          className="z-10 font-semibold rounded-full px-8 py-2 leading-normal bg-transparent border border-grey text-grey hover:border-bby-blue hover:bg-bby-blue hover:text-white m-6"
        >
          Sign In
        </button>
      </Link>
    </div>
    <div
      className="hero-img h-screen w-screen absolute pin"
    />
    <div
      className="kinda-center z-10 w-full"
    >
      <h1
        className="text-white italic text-center leading-none whitespace-no-wrap text-5xl md:text-massive tracking-wide mb-2"
      >
        Quantified
      </h1>
      <h2
        className="text-white italic text-center leading-none text-xl font-thin"
      >
        A streamlined system for efficiently and accurately managing retail inventory.
      </h2>
    </div>
  </section>
  <section
    className="bg-bby-blue"
    id="how-section"
  >
    <h1
      className="text-4xl pt-6 text-white text-center"
    >
      features
    </h1>
    <div
      className="card-container flex flex-wrap justify-center trans-300ms-all"
    >
      <div
        className="how-card flex flex-col items-center justify-center h-64 w-64 m-8 bg-white rounded"
      >
        <svg
          enableBackground="new 0 0 96 96"
          version="1.1"
          viewBox="0 0 96 120"
          x="0"
          xmlns="http://www.w3.org/2000/svg"
          y="0"
        >
          <g>
            <path
              d="M82.8,28.1c-2.9-4.9-10.9-6-21.1-3.6c-3-9.9-8-16.2-13.7-16.2s-10.7,6.3-13.7,16.2c-10.2-2.3-18.2-1.2-21.1,3.6 c-1.2,2.1-1.4,4.5-0.7,7.4c1,3.8,3.9,8.2,8,12.5c-4.2,4.3-7,8.7-8,12.5c-0.8,2.9-0.5,5.3,0.7,7.4c1.9,3.3,6.1,4.9,11.7,4.9 c2.8,0,6-0.4,9.4-1.2c3,9.9,8,16.2,13.7,16.2s10.7-6.3,13.7-16.2c3.4,0.8,6.6,1.2,9.4,1.2c5.6,0,9.8-1.6,11.7-4.9 c1.2-2.1,1.4-4.5,0.7-7.4c-1-3.8-3.9-8.2-8-12.5c4.2-4.3,7-8.7,8-12.5C84.3,32.6,84.1,30.1,82.8,28.1z M48,11.7 c4.3,0,7.9,5.7,10.3,13.6c-3.3,1-6.8,2.3-10.3,3.8c-3.6-1.6-7-2.9-10.3-3.8C40.1,17.3,43.7,11.7,48,11.7z M61.5,48 c0,2.5-0.1,5-0.3,7.5c-2,1.4-4.2,2.7-6.4,4c-2.2,1.2-4.4,2.4-6.8,3.5c-2.3-1-4.6-2.3-6.8-3.5c-2.3-1.3-4.3-2.6-6.4-4 c-0.3-2.5-0.3-5-0.3-7.5c0-2.5,0.1-5,0.3-7.5c2-1.4,4.2-2.7,6.4-4c2.2-1.2,4.4-2.4,6.8-3.5c2.3,1,4.6,2.3,6.8,3.5 c2.3,1.3,4.3,2.6,6.4,4C61.4,43,61.5,45.5,61.5,48z M64.9,43.1c2.1,1.6,4,3.2,5.7,4.9c-1.7,1.7-3.6,3.3-5.7,4.9 c0.1-1.6,0.1-3.2,0.1-4.9C64.9,46.3,64.9,44.7,64.9,43.1z M60.6,60c-0.3,2.5-0.8,5-1.4,7.2c-2.3-0.7-4.6-1.5-7-2.4 c1.4-0.7,2.8-1.5,4.3-2.3C57.9,61.7,59.3,60.9,60.6,60z M43.7,64.8c-2.3,1-4.7,1.7-7,2.4c-0.5-2.3-1-4.8-1.4-7.2 c1.4,0.9,2.7,1.7,4.2,2.5C41,63.4,42.4,64.1,43.7,64.8z M31.1,52.9c-2.1-1.6-4-3.2-5.7-4.9c1.7-1.7,3.6-3.3,5.7-4.9 c-0.1,1.6-0.1,3.2-0.1,4.9C31.1,49.7,31.1,51.3,31.1,52.9z M35.4,36c0.3-2.5,0.8-5,1.4-7.2c2.3,0.7,4.6,1.5,7,2.4 c-1.4,0.7-2.8,1.5-4.3,2.3C38.1,34.3,36.7,35.1,35.4,36z M56.4,33.5c-1.4-0.8-2.8-1.6-4.3-2.3c2.3-1,4.7-1.7,7-2.4 c0.5,2.3,1,4.8,1.4,7.2C59.3,35.1,57.9,34.3,56.4,33.5z M16.1,29.8c1.3-2.3,4.3-3.2,8.3-3.2c2.6,0,5.6,0.4,8.9,1.2 c-0.8,3.3-1.4,6.9-1.8,10.8c-3.2,2.3-6.1,4.6-8.6,7c-3.6-3.9-6.2-7.7-7.1-10.9C15.3,32.6,15.4,31.1,16.1,29.8z M16.1,66.2 c-0.7-1.2-0.8-2.8-0.3-4.8c0.9-3.3,3.4-7.1,7.1-10.9c2.5,2.3,5.4,4.7,8.6,7c0.3,3.8,1,7.5,1.8,10.8C25.1,70.2,18.3,69.9,16.1,66.2z M48,84.3c-4.3,0-7.9-5.7-10.3-13.6c3.3-1,6.8-2.3,10.3-3.8c3.6,1.6,7,2.9,10.3,3.8C55.9,78.7,52.3,84.3,48,84.3z M79.9,66.2 c-2.2,3.7-8.9,3.9-17.2,2c0.8-3.3,1.4-6.9,1.8-10.8c3.2-2.3,6.1-4.6,8.6-7c3.6,3.9,6.2,7.7,7.1,10.9C80.7,63.4,80.6,64.9,79.9,66.2 z M80.1,34.6c-0.9,3.3-3.4,7.1-7.1,10.9c-2.5-2.3-5.4-4.7-8.6-7c-0.3-3.8-1-7.5-1.8-10.8c3.3-0.8,6.3-1.2,8.9-1.2c4,0,7,1,8.3,3.2 C80.6,31.1,80.7,32.6,80.1,34.6z"
            />
            <path
              d="M48,41.5c-3.8,0-7,3-7,6.5s3.1,6.5,7,6.5s7-3,7-6.5S51.8,41.5,48,41.5z M48,51c-1.9,0-3.5-1.4-3.5-3s1.6-3,3.5-3 s3.5,1.4,3.5,3S49.9,51,48,51z"
            />
          </g>
        </svg>
        <h3>
          Simple
        </h3>
        <p>
          ...but powerful
        </p>
      </div>
      <div
        className="how-card flex flex-col items-center justify-center h-64 w-64 m-8 bg-white rounded"
      >
        <svg
          enableBackground="new 0 0 100 100"
          version="1.1"
          viewBox="0 0 100 125"
          x="0"
          xmlns="http://www.w3.org/2000/svg"
          y="0"
        >
          <path
            d="M73.759,31.69L84.49,52.111c0,0-1.179,2.614-3.949,5.59c-2.013,2.16-4.621,4.024-4.621,4.024L56.169,39.815 c0,0-0.818-0.895-2.16-1.044c-0.988-0.11-2.832,0.596-2.832,0.596l-11.179,6.112c0,0-2.612,1.816-4.472-0.819 c-1.789-2.534,1.416-4.77,1.416-4.77s10.062-8.272,16.694-11.329c3.308-1.524,6.335-0.969,8.422,0.149s5.443,3.471,7.081,4.1 C71.075,33.554,73.759,31.69,73.759,31.69z"
          />
          <path
            d="M94.354,48.29L82.518,26.224c-0.312-0.581-1.035-0.799-1.615-0.487l-4.576,2.415c-0.58,0.312-0.798,1.035-0.487,1.616 l11.838,22.065c0.312,0.58,1.034,0.799,1.615,0.488l4.575-2.416C94.448,49.594,94.666,48.87,94.354,48.29z M89.781,48.571 c-0.927,0-1.676-0.75-1.676-1.676c0-0.926,0.749-1.678,1.676-1.678c0.926,0,1.678,0.752,1.678,1.678 C91.459,47.82,90.707,48.571,89.781,48.571z"
          />
          <path
            d="M26.31,60.463c-0.072-0.039-1.797-0.993-4.822-3.714c-1.76-1.582-3.682-4.366-3.763-4.484 c-0.252-0.366-0.279-0.843-0.07-1.235l10.632-19.975c0.284-0.531,0.922-0.768,1.484-0.55l0,0c0.001,0,0.571,0.21,1.881,0.461 c1.346,0.259,2.386-0.047,4.109-0.557c0.339-0.1,0.698-0.207,1.082-0.313c1.006-0.285,2.049-0.867,3.154-1.484 c1.311-0.732,2.666-1.489,4.176-1.878c3.313-0.852,5.315,1.569,5.398,1.673l-1.848,1.508c-0.046-0.054-1.153-1.334-2.956-0.872 c-1.212,0.312-2.429,0.993-3.607,1.65c-1.178,0.658-2.395,1.338-3.671,1.697c-0.373,0.105-0.722,0.208-1.052,0.306 c-1.855,0.548-3.319,0.98-5.234,0.612c-0.514-0.098-0.932-0.193-1.265-0.277l-9.83,18.465c0.631,0.864,1.874,2.487,2.974,3.479 c2.82,2.534,4.355,3.383,4.37,3.392L26.31,60.463z"
          />
          <path
            d="M58.998,78.492c-1.516,0-2.521-0.875-2.636-0.978l0.801-0.884l0.788-0.895c0.312,0.266,1.453,0.993,2.98-0.778 c1.563-1.813,0.364-3.692,0.353-3.71l1.995-1.307c0.766,1.169,1.637,4.047-0.542,6.574c-1.061,1.229-2.237,1.892-3.5,1.969 C59.156,78.489,59.077,78.492,58.998,78.492z"
          />
          <path
            d="M66.146,73.606c-1.14,0-2.076-0.434-2.575-0.662c-1.127-0.521-2.87-2.368-3.207-2.732c-0.447-0.482-0.419-1.238,0.064-1.685 c0.483-0.448,1.237-0.42,1.685,0.064c0.821,0.885,1.993,1.973,2.458,2.187c1.078,0.497,2.069,0.813,3.29-0.37 c1.503-1.455,0.218-3.419-0.015-3.745l-6.881-7.327c-0.45-0.479-0.427-1.234,0.053-1.684c0.479-0.451,1.234-0.428,1.686,0.052 l6.931,7.378c0.03,0.031,0.057,0.063,0.082,0.099c1.02,1.348,2.29,4.532-0.195,6.94C68.344,73.263,67.162,73.606,66.146,73.606z"
          />
          <path
            d="M73.405,69.648c-1.737,0-3.145-0.818-3.825-1.472l-3.661-3.587l1.668-1.704l3.653,3.577 c0.165,0.152,1.838,1.597,4.077,0.212c0.929-0.575,1.495-1.211,1.639-1.836c0.142-0.615-0.153-1.091-0.166-1.111l1.974-1.34 c0.09,0.13,0.866,1.311,0.533,2.906c-0.272,1.312-1.189,2.457-2.725,3.408C75.459,69.392,74.38,69.648,73.405,69.648z"
          />
          <path
            d="M32.292,65.333c-1.208,1.209-3.166,1.209-4.375,0l0,0c-1.207-1.208-1.207-3.166,0-4.373l1.476-1.476 c1.208-1.209,3.166-1.209,4.374,0l0,0c1.208,1.208,1.208,3.164,0,4.373L32.292,65.333z"
          />
          <path
            d="M38.329,70.302c-1.207,1.209-3.166,1.209-4.374,0l0,0c-1.207-1.207-1.207-3.166,0-4.374l4.849-4.849 c1.207-1.207,3.166-1.207,4.374,0l0,0c1.208,1.209,1.207,3.167,0,4.373L38.329,70.302z"
          />
          <path
            d="M45.259,75.022c-1.208,1.207-3.166,1.207-4.374,0l0,0c-1.208-1.209-1.208-3.168,0-4.375l6.152-6.153 c1.208-1.206,3.167-1.206,4.374,0l0,0c1.208,1.209,1.208,3.167,0,4.375L45.259,75.022z"
          />
          <path
            d="M52.44,78.996c-1.209,1.208-3.167,1.208-4.375,0l0,0c-1.208-1.207-1.208-3.165,0-4.373l3.209-3.209 c1.207-1.207,3.166-1.208,4.373,0l0,0c1.208,1.208,1.208,3.166,0,4.374L52.44,78.996z"
          />
          <circle
            cx="11.147"
            cy="47.152"
            r="1.677"
          />
          <g>
            <path
              d="M19.224,27.351c0.19,0,0.384,0.046,0.563,0.143l2.925,1.643c0.58,0.312,0.797,1.036,0.484,1.615L12.965,50.137 c-0.215,0.398-0.625,0.626-1.05,0.626c-0.19,0-0.385-0.047-0.565-0.143l-2.924-1.643c-0.581-0.312-0.797-1.036-0.485-1.616 l10.231-19.383C18.388,27.579,18.799,27.351,19.224,27.351 M19.224,24.966c-1.319,0-2.525,0.721-3.151,1.881l-0.005,0.009 l-0.005,0.01L5.836,46.241c-0.449,0.838-0.545,1.802-0.272,2.715c0.272,0.906,0.877,1.654,1.707,2.106l2.912,1.636L10.2,52.71 l0.019,0.01c0.521,0.278,1.106,0.427,1.696,0.427c1.318,0,2.525-0.72,3.15-1.88l0.005-0.01l0.005-0.01l10.227-19.375 c0.923-1.727,0.28-3.882-1.436-4.823l-2.911-1.635l-0.018-0.01l-0.018-0.01C20.399,25.115,19.812,24.966,19.224,24.966 L19.224,24.966z"
            />
          </g>
          <path
            d="M62.58,71.984c-0.317,0-0.635-0.126-0.869-0.376l-6.932-7.377c-0.45-0.481-0.428-1.236,0.053-1.686 c0.48-0.451,1.235-0.428,1.686,0.052l6.931,7.38c0.451,0.479,0.428,1.234-0.052,1.684C63.166,71.878,62.872,71.984,62.58,71.984z"
          />
        </svg>
        <h3>
          Reliable
        </h3>
        <p>
          There when you need it
        </p>
      </div>
      <div
        className="how-card flex flex-col items-center justify-center h-64 w-64 m-8 bg-white rounded"
      >
        <svg
          version="1.1"
          viewBox="0 0 48 60.00000125"
          x="0px"
          y="0px"
        >
          <g
            transform="translate(0,-1004.3622)"
          >
            <g
              fill="#000000"
            >
              <path
                d="m 31.410156,1031.2812 a 0.75007506,0.75007506 0 0 0 -0.443359,0.1739 l -9.527344,7.9433 -3.958984,-4.5371 a 0.75007506,0.75007506 0 0 0 -1.3125,0.4317 l -0.361328,4.4824 a 0.75007506,0.75007506 0 0 0 0.18164,0.5547 l 4.441407,5.0898 a 0.75007506,0.75007506 0 0 0 1.044921,0.084 l 10.089844,-8.4141 a 0.75007506,0.75007506 0 0 0 0.267578,-0.5156 l 0.361328,-4.4824 a 0.75007506,0.75007506 0 0 0 -0.783203,-0.8106 z m -0.855468,2.4688 -0.19336,2.3906 -9.283203,7.7422 -3.751953,-4.3008 0.193359,-2.3945 3.271485,3.75 a 0.75007506,0.75007506 0 0 0 1.044922,0.084 l 8.71875,-7.2715 z"
              />
              <path
                d="m 38,1050.8613 c 1.859441,0 3.380859,-1.5194 3.380859,-3.3789 l 0,-19.2851 c 0,-1.8595 -1.521418,-3.3789 -3.380859,-3.3789 l -28,0 c -1.8594411,0 -3.3808594,1.5194 -3.3808594,3.3789 l 0,19.2851 c 0,1.8595 1.5214183,3.3789 3.3808594,3.3789 l 28,0 z m 0,-1.5 -28,0 c -1.0543821,0 -1.8808594,-0.8245 -1.8808594,-1.8789 l 0,-19.2851 c 0,-1.0544 0.8264773,-1.8789 1.8808594,-1.8789 l 28,0 c 1.054383,0 1.880859,0.8245 1.880859,1.8789 l 0,19.2851 c 0,1.0544 -0.826476,1.8789 -1.880859,1.8789 z"
              />
              <path
                d="m 23.462891,1005.8613 a 0.75007494,0.75007494 0 0 0 -0.25,0.049 c -6.359502,0.1356 -11.5,5.0907 -11.5,11.2871 l 0,7.8886 a 0.75007494,0.75007494 0 1 0 1.5,0 l 0,-7.8886 c 0,-5.4456 4.543019,-9.836 10.25,-9.836 l 1.074218,0 c 5.706981,0 10.25,4.3904 10.25,9.836 l 0,8.1465 a 0.75007494,0.75007494 0 1 0 1.5,0 l 0,-8.1465 c 0,-6.2823 -5.268378,-11.336 -11.75,-11.336 l -1.074218,0 z m 0.908203,2.9746 c -3.583478,0 -6.010747,1.1429 -7.515625,2.7383 -1.504879,1.5954 -2.074547,3.5761 -2.197266,5.1973 a 0.75007494,0.75007494 0 0 0 -0.002,0.057 l 0,8.459 a 0.75007494,0.75007494 0 1 0 1.5,0 l 0,-8.4121 c 0.104569,-1.3602 0.587082,-2.9951 1.791016,-4.2715 1.206633,-1.2792 3.15083,-2.2676 6.423828,-2.2676 4.00731,0 7.388672,2.95 7.388672,6.4922 l 0,8.4961 a 0.75007494,0.75007494 0 1 0 1.5,0 l 0,-8.4961 c 0,-4.4802 -4.143408,-7.9922 -8.888672,-7.9922 z"
              />
            </g>
          </g>
        </svg>
        <h3>
          Safe
        </h3>
        <p>
          Without compromises
        </p>
      </div>
      <div
        className="how-card flex flex-col items-center justify-center h-64 w-64 m-8 bg-white rounded"
      >
        <svg
          enableBackground="new 0 0 100 100"
          version="1.1"
          viewBox="0 0 100 125"
          x="0"
          xmlns="http://www.w3.org/2000/svg"
          y="0"
        >
          <g>
            <path
              d="M49.876,33.14c2.979,0,5.401-2.423,5.401-5.401s-2.423-5.402-5.401-5.402s-5.401,2.423-5.401,5.402 S46.897,33.14,49.876,33.14z M49.045,24.451c0.104,0.441,0.482,0.775,0.955,0.775c1.385,0,2.512,1.127,2.512,2.512 S51.385,30.25,50,30.25c-0.473,0-0.85,0.334-0.955,0.775c-1.473-0.373-2.571-1.699-2.571-3.287 C46.475,26.15,47.572,24.824,49.045,24.451z"
            />
            <path
              d="M53.822,39.208c0-2.176-1.771-3.946-3.946-3.946s-3.946,1.771-3.946,3.946s1.771,3.946,3.946,3.946 S53.822,41.384,53.822,39.208z M47.93,39.208c0-0.889,0.603-1.633,1.419-1.864c0.118,0.42,0.488,0.735,0.946,0.735 c0.623,0,1.13,0.507,1.13,1.13c0,0.623-0.507,1.129-1.13,1.129c-0.459,0-0.828,0.315-0.947,0.735 C48.532,40.841,47.93,40.097,47.93,39.208z"
            />
            <path
              d="M50.876,62.793v-5.223h3.438l8.759,6.417v-18.01l-4.769-1.837c0.275-2.748,0.422-5.661,0.422-8.655 c0-14.802-7.688-26.621-8.016-27.116l-0.835-1.265l-0.835,1.265c-0.327,0.496-8.016,12.314-8.016,27.116 c0,2.993,0.147,5.904,0.422,8.65l-4.77,1.838v18.01l8.753-6.412h3.447v5.223H50.876z M61.073,47.351v12.692l-5.229-3.83 c0.981-2.767,1.738-6.193,2.228-10.018L61.073,47.351z M49.876,10.87c0.776,1.373,1.987,3.701,3.177,6.692h-6.354 C47.889,14.571,49.1,12.242,49.876,10.87z M38.677,47.346l3.003-1.157c0.49,3.825,1.246,7.251,2.227,10.018l-5.23,3.831V47.346z M45.799,55.571c-1.767-4.963-2.773-12.226-2.773-20.085c0-5.978,1.354-11.506,2.923-15.923h7.854 c1.57,4.417,2.923,9.945,2.923,15.923c0,7.859-1.007,15.122-2.773,20.085h-3.077V44.208h-2v11.362H45.799z"
            />
            <path
              d="M76.203,79.707c-1.628,0-3.219,0.478-4.574,1.359c-0.314-0.342-0.656-0.658-1.024-0.944 c-1.105-3.446-4.337-5.824-7.979-5.824c-1.058,0-2.091,0.199-3.077,0.591c-1.537-0.827-2.336-1.36-2.886-3.677 c-0.657-2.773-0.645-9.356-0.645-9.423c0.001-0.266-0.116-0.521-0.304-0.709c-0.392-0.393-1.11-0.369-1.474,0.048 c-0.454,0.52-0.989,0.963-1.589,1.316c-0.25,0.147-0.421,0.394-0.475,0.672c-0.245-0.148-0.545-0.183-0.819-0.097 c-0.339,0.108-0.694,0.188-1.056,0.237c-0.259,0.036-0.482,0.167-0.638,0.356c-0.167-0.177-0.398-0.292-0.659-0.31 c-0.353-0.023-0.708-0.078-1.057-0.161c-0.27-0.065-0.552-0.014-0.779,0.135c-0.082-0.255-0.265-0.472-0.515-0.593 c-0.529-0.258-1.019-0.584-1.453-0.97c-0.304-0.27-0.742-0.328-1.104-0.148c-0.364,0.179-0.584,0.561-0.557,0.966 c0.111,1.589,0.265,5.592-0.686,8.446c-0.892,2.671-1.933,4.29-2.402,4.938c-0.298,0.005-0.659,0.011-0.844,0.011 c-3.964,0-7.294,2.74-8.159,6.472c-2.166-0.958-4.772-0.939-6.95,0.083c-0.524-0.112-1.048-0.169-1.562-0.169 c-4.151,0-7.528,3.377-7.528,7.527c0,0.475,0.046,0.955,0.137,1.428c0.091,0.471,0.503,0.811,0.982,0.811H82.96 c0.396,0,0.754-0.233,0.914-0.595c0.476-1.074,0.717-2.214,0.717-3.39C84.591,83.47,80.828,79.707,76.203,79.707z M82.274,90.08 h-64.86c-0.003-0.08-0.005-0.159-0.005-0.239c0-3.511,3.309-6.3,6.953-5.334c0.241,0.063,0.502,0.035,0.724-0.082 c0.298-0.158,0.609-0.282,0.925-0.388c-0.56,0.523-0.985,1.176-1.207,1.929c-0.156,0.53,0.146,1.086,0.676,1.243 c0.531,0.156,1.085-0.146,1.242-0.675c0.178-0.602,0.579-1.099,1.131-1.398c0.553-0.3,1.187-0.368,1.788-0.189 c1.242,0.368,1.954,1.677,1.587,2.919c-0.156,0.53,0.146,1.086,0.676,1.243c0.095,0.028,0.189,0.042,0.283,0.042 c0.432,0,0.83-0.282,0.959-0.717c0.368-1.246,0.135-2.519-0.507-3.541c0.015-0.006,0.03-0.008,0.045-0.015 c0.325-0.162,0.537-0.487,0.554-0.85c0.154-3.419,2.953-6.098,6.372-6.098c0.356,0,1.36-0.02,1.36-0.02 c0.278-0.006,0.542-0.127,0.728-0.336c0.069-0.079,1.726-1.972,3.056-5.96c0.578-1.736,0.794-3.803,0.853-5.603 c0.32,0.398,0.813,0.653,1.362,0.653c0.549,0,1.039-0.254,1.36-0.651c0.321,0.396,0.812,0.651,1.36,0.651s1.04-0.254,1.361-0.651 c0.32,0.397,0.812,0.651,1.36,0.651c0.793,0,1.465-0.531,1.679-1.256c0.08,2.125,0.255,4.693,0.629,6.269 c0.787,3.323,2.271,4.116,4.149,5.12l0.162,0.086c0.278,0.149,0.61,0.159,0.896,0.024c0.859-0.402,1.768-0.606,2.7-0.606 c2.666,0,5.024,1.684,5.951,4.145c-0.644-0.112-1.306-0.094-1.95,0.095c-2.303,0.67-3.631,3.089-2.961,5.391 c0.127,0.437,0.526,0.721,0.96,0.721c0.092,0,0.187-0.013,0.279-0.041c0.53-0.154,0.835-0.709,0.681-1.239 c-0.361-1.244,0.356-2.55,1.6-2.913c0.603-0.175,1.237-0.106,1.788,0.197c0.55,0.302,0.949,0.8,1.125,1.403 c0.153,0.529,0.702,0.833,1.239,0.681c0.53-0.154,0.835-0.709,0.681-1.24c-0.021-0.073-0.059-0.137-0.083-0.208 c0.055-0.031,0.117-0.048,0.165-0.089c1.149-0.964,2.606-1.496,4.104-1.496c3.522,0,6.388,2.866,6.388,6.388 C82.591,88.776,82.484,89.441,82.274,90.08z"
            />
          </g>
        </svg>
        <h3>
          Fast
        </h3>
        <p>
          Designed with speed in mind
        </p>
      </div>
    </div>
  </section>
  <section
    id="about-section"
  >
    <h1>
      About Quantified
    </h1>
    <blockquote>
      "A streamlined system for efficiently and accurately managing retail inventory."
    </blockquote>
    <p>
      That sounds great and all, but what does that mean exactly? Imagine for a moment that you are in a retail setting. When shipments of products come into your store, they will be accompanied by a manifest that notates exactly what products and how many of them were put onto the trailer when it left the distribution center. It is the job of one employee to verify everything listed on the manifest actually makes it to the store, not an easy or quick task by any means.
    </p>
    <p>
      The previous workflow (at least at Best Buy) for counting all of these products involved printing out a copy of the aforementioned manifest, then tallying up the products one by one. Sounds monotonous and error prone right? That's where Quantified comes in to save the day by bearing the responsibility of accounting for everything.
    </p>
    <p>
      Instead of manually counting each product, Quantified allows the user to simply scan the bar code of the product and automatically populates all of its relevant information into a row in a table. From there, after everything has been scanned, the table can be formatted to mirror the manifest and printed for seamless comparison with the manifest for the final verification.
    </p>
    <p>
      Additionally, Quantified is built with battle tested and industry leading technology as well as the Best Buy API. All together this yields unmatched dependability, speed, and security.
    </p>
  </section>
  <footer>
    <ul>
      <li>
        <Link
          replace={false}
          to="/sign-up"
        >
          Start here
        </Link>
      </li>
      <li>
        <Link
          replace={false}
          to="/sign-in"
        >
          Sign in
        </Link>
      </li>
      <li>
        <a
          href="https://developer.bestbuy.com/api-status"
          target="_blank"
        >
          Official Best Buy API Status
        </a>
      </li>
      <li>
        <a
          href="https://github.com/chrstntdd"
          target="_blank"
        >
          Created by Christian Todd
        </a>
      </li>
    </ul>
  </footer>
</div>
`;
